tests/mesquite/basic_01.cc

```diff
@@ -0,0 +1,80 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2018 by the deal.II authors
+ *
+ * This file is part of the deal.II library.
+ *
+ * The deal.II library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE at
+ * the top level of the deal.II distribution.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+/**
+ * This demonstrates the application of a high-level API, namely the Laplace
+ * smoother, to a distorted grid.
+ *
+ * The tested mesh is courtesy of the Mesquite project.
+ * This simple program is part of a tutorial found in the user guide.
+ *
+ * Knupp, P.; Freitag-Diachin, L. & Tidwell, B.
+ * Mesquite Mesh Quality Improvement Toolkit User's Guide
+ * Sandia National Laboratories, Sandia National Laboratories, 2013
+ *
+ * See sec 3.1.3 Improving the Mesh with a Wrapper Class
+ */
+
+
+#include "../tests.h"
+
+#include <deal.II/mesquite/mesquite.h>
```

Comment:
i think you forgot to add this one, I can't find it

Reply:
Indeed I did. I've now added this file.